<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NovoUsuario</class>
 <widget class="QDialog" name="NovoUsuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Novo Usuário</string>
  </property>
  <widget class="QDialogButtonBox" name="ok_cancela_botao">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>320</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="videoCapture">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>32</y>
     <width>282</width>
     <height>240</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>280</y>
     <width>137</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="group_numFotos">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QPushButton" name="tiraFoto_botao">
      <property name="text">
       <string>Tirar Foto</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="num_fotos_label">
      <property name="text">
       <string>0/10</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>24</y>
     <width>401</width>
     <height>217</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="group_Form">
    <item>
     <layout class="QHBoxLayout" name="group_matricula">
      <property name="spacing">
       <number>24</number>
      </property>
      <item>
       <widget class="QLabel" name="matricula_label">
        <property name="text">
         <string>Matrícula: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="matricula_field"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="group_tipo_pessoa">
      <item alignment="Qt::AlignLeft">
       <widget class="QLabel" name="tipo__label">
        <property name="text">
         <string>Tipo :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="tipo_pessoa_field">
        <property name="minimumSize">
         <size>
          <width>312</width>
          <height>22</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Selecione</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Professor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Aluno</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Servidor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Externo</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="group_nome">
      <property name="spacing">
       <number>44</number>
      </property>
      <item>
       <widget class="QLabel" name="nome_label">
        <property name="text">
         <string>Nome :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nome_field"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="group_sobrenome">
      <property name="spacing">
       <number>11</number>
      </property>
      <item>
       <widget class="QLabel" name="sobrenome_label">
        <property name="text">
         <string>Sobrenome :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sobrenome_field"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="group_materias">
      <property name="spacing">
       <number>28</number>
      </property>
      <item>
       <widget class="QLabel" name="materias_label">
        <property name="text">
         <string>Matérias :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="materia_field"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="erro_label">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>264</y>
     <width>273</width>
     <height>33</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>ok_cancela_botao</zorder>
  <zorder>videoCapture</zorder>
  <zorder>erro_label</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok_cancela_botao</sender>
   <signal>rejected()</signal>
   <receiver>NovoUsuario</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
